src/carousel.coffee

```diff
@@ -81,6 +81,11 @@ Ember.Widgets.CarouselComponent = Ember.Component.extend
         $active.removeClass(['active', direction].join(' '))
         @set 'sliding', no
 
+        # Send an action out to signal that the sliding transition has ended
+        # It can is useful to avoid having observers in any parentView having
+        # this component.
+        @send 'transitionEnded'
```

Comment:
I don't think you need the observer comment as it is not specific to the implementation, more of a general concept around observers/actions.